pkg/remotewrite/config.go

```diff
@@ -237,6 +237,19 @@ func parseEnvs(env map[string]string) (Config, error) {
 		c.Headers[k] = v
 	}
 
+	if headers, headersDefined := env["K6_PROMETHEUS_RW_HTTP_HEADERS"]; headersDefined {
+		if c.Headers == nil {
+			c.Headers = make(map[string]string)
```

Comment:
How about moving this into the initialization to the top by default:

```diff
-var c Config
+c := Config{Headers: make(map[string]string)}
```

There is no significant difference in efficiency, but the code's reader will be a bit happier without 
`if c.Headers == nil`